packages/edit-post/src/components/layout/index.js

```diff
@@ -129,6 +131,9 @@ function Layout( { styles } ) {
 			),
 			// translators: Default label for the Document in the Block Breadcrumb.
 			documentLabel: postTypeLabel || _x( 'Document', 'noun' ),
+			isListViewOpenByDefault: select( editPostStore ).isFeatureActive(
```

Comment:
@talldan 

Do you know if the post editor preferences modal is, or should be, migrated over to the preferences store?  Done for the site editor in https://github.com/WordPress/gutenberg/pull/39485

I'm happy to take a look if you think it's worth it.

Reply:
@ramonjd Do you mean the component or the preferences data?

The situation is:
- There's a generic preferences modal component in the interface package and that's used in the post and site editors.
- All preferences data is in the preferences package for those editors (`select( editPostStore ).isFeatureActive` is a registry selector that calls through to the preferences store).

So I think you're all good. At some point I'll look at deprecating the post editor's `isFeatureActive` selector.

Reply:
> Do you mean the component or the preferences data?

Ah I was looking at the [EnableFeature component](https://github.com/WordPress/gutenberg/blob/d10329149ce45ef3dd9572fb0a03ebff548cc092/packages/edit-post/src/components/preferences-modal/options/enable-feature.js). It uses `isFeatureActive`.

I'm just comparing it with the corresponding [EnableFeature component](https://github.com/WordPress/gutenberg/blob/8d0a84f7edaa090acb26d7d30c500ace30da0a45/packages/edit-site/src/components/preferences-modal/enable-feature.js) for the site editor preferences, which uses the preferences store.

> (select( editPostStore ).isFeatureActive is a registry selector that calls through to the preferences store)

Great, thanks for confirming. 

I was mainly checking to see if how I'm selecting the value of the preference for the [post](https://github.com/WordPress/gutenberg/pull/40757/files#diff-8933c0b377d83066f7dde9c8b573bab9a7bfe60f73919e4e5e80aff92a88800cR134) and [site](https://github.com/WordPress/gutenberg/pull/40757/files#diff-53f39da7a5f1f996f5359a2dd7b2747fb3d60a494523dd73dba946fab8e82592R118) editors is okay for now. 

I mean, it works 😄